stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -135,9 +137,72 @@ public static Source createSourceOnServer(
             InvalidRequestException,
             APIConnectionException,
             APIException {
+        return createSourceOnServer(
+                uidProvider,
+                context,
+                sourceParams,
+                publishableKey,
+                null,
+                loggingResponseListener);
+    }
+
+    /**
+     * Create a {@link Source} using the input {@link SourceParams}.
+     *
+     * @param uidProvider a provider for UUID items in test
+     * @param context a {@link Context} object for aquiring resources
+     * @param sourceParams a {@link SourceParams} object with {@link Source} creation params
+     * @param publishableKey an API key
+     * @param stripeAccount a connected Stripe Account ID
+     * @param loggingResponseListener a listener for logging responses
+     * @return a {@link Source} if one could be created from the input params,
+     * or {@code null} if not
+     * @throws AuthenticationException if there is a problem authenticating to the Stripe API
+     * @throws InvalidRequestException if one or more of the parameters is incorrect
+     * @throws APIConnectionException if there is a problem connecting to the Stripe API
+     * @throws APIException for unknown Stripe API errors. These should be rare.
+     */
+    @Nullable
+    public static Source createSourceOnServer(
```

Comment:
This seems different than the iOS version and I'm not sure if that's just because the Android API is different or if I'm reading this wrong. Does your version only work for sources? I guess our source creation API doesn't have the ability to set publishableKey/stripeAccount on a per-request basis, we only have the one on the Stripe object itself.